src/main/java/in/balendin/schoolmanagment/pages/AddOrganizationPage.java

```diff
@@ -0,0 +1,185 @@
+package in.balendin.schoolmanagment.pages;
+
+import in.balendin.schoolmanagment.models.OrganizationData;
+import net.serenitybdd.core.pages.WebElementFacade;
+import net.thucydides.core.pages.PageObject;
+import org.junit.Assert;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.support.FindBy;
+
+public class AddOrganizationPage extends PageObject {
+
+    @FindBy(xpath = "/html/body/div/aside/div/nav/ul/li[1]/ul/li[3]/a")
```

Comment:
@rajaYendluri Please  use the relative Xpath for all the form elements

Reply:
completed